templates/evaluations/components/auto_population_rules_panel.html

```
@@ -0,0 +1,61 @@
+                 href="{% url 'evaluations:auto_population_rule_edit' request.team.slug dataset.id rule.id %}">
+                {% translate "Edit" %}
+              </a>
+              <form method="post"
+                    action="{% url 'evaluations:auto_population_rule_toggle' request.team.slug rule.id %}"
+                    class="inline">
+                {% csrf_token %}
+                <button type="submit" class="link">
+                  {% if rule.is_enabled %}{% translate "Disable" %}{% else %}{% translate "Enable" %}{% endif %}
+                </button>
+              </form>
+              <form method="post"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)